frontend/.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:storybook/recommended"
+  ],
+  "overrides": [],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": ["react", "@typescript-eslint"],
+  "rules": {
+    "react/no-unknown-property": ["error", { "ignore": ["css"] }],
+    "react/react-in-jsx-scope": "off",
+    "react/self-closing-comp": ["error", { "component": true, "html": true }],
+    "@typescript-eslint/consistent-type-imports": [
+      "error",
+      {
+        "prefer": "type-imports"
+      }
+    ]
+  }
+}
```

Comment:
왜 갑자기 json?

Reply:
Module 부분이 오류가 나서 그냥 없애버림 😂

Reply:
혹시 왜 오류가 나는지 확인해보셨나요?!

Reply:
```
File is a CommonJS module; it may be converted to an ES module.ts(80001)
'module' is not defined.eslint[no-undef](https://eslint.org/docs/latest/rules/no-undef)
``` 
이런 오류가 나는 건 이전에도 똑같았는데, 현재 브랜치에서 `"react/no-unknown-property": ["error", { "ignore": ["css"] }]` 이 룰 하나를 추가하니 commit이 안 되더라구요. 
이유는 eslint 오류가 있어서 husky가 commit을 허용해주지 않은 거였습니다. (수정이 발생할 때 husky가 eslint 오류를 잡아내서) 이 부분만 eslint 에러 넘기는 주석을 넣어도 되지만, 주석 대신 json으로 바꾸는 방향으로 개선했습니다!

Reply:
아하